os_migrate/plugins/module_utils/network.py

```diff
@@ -239,7 +239,10 @@ def serialize_security_group(sdk_sec):
     """
     expected_type = openstack.network.v2.security_group.SecurityGroup
     if type(sdk_sec) != expected_type:
-        raise exc.UnexpectedResourceType(expected_type, type(sdk_sec))
```

Comment:
To be discussed

Reply:
I found the problem. We should not use methods directly on conn, that's old API:

https://github.com/os-migrate/os-migrate/blob/5fb9018574e082e48b80d523ef009b1d32febef2/os_migrate/plugins/modules/export_security_group.py#L80

We have to use methods under `conn.network`, in this case probably  `find_security_group` with `ignore_missing=False`.

Reply:
ack, so the old api call is returning a Munch object instead of the one we need (SecurityGroup).